UploadDaemon_Test/SymbolAnalysis/SymbolCollectionResolverTest.cs

```diff
@@ -10,56 +10,113 @@ namespace UploadDaemon.SymbolAnalysis
     [TestFixture]
     class SymbolCollectionResolverTest
     {
-        private readonly GlobPatternList someAssemblyPatterns = new GlobPatternList(new List<string> { "*" }, new List<string> { });
-        private readonly string testSymbolFileName = $"{TestUtils.TestDataDirectory}\\Some.pdb";
+        private static readonly string TestDataDirectory = $"{TestUtils.TestDataDirectory}\\Tmp";
```

Comment:
I'd prefer using the system temp dir if possible. otherwise you may get a dirty workspace if running the tests aborts, which is annoying during local development

Reply:
if that's not easily possible with .NET, however, it's OK to leave it as is

Reply:
however, i'd rename it to `TestTempDirectory` to make the difference between this and `TestUtils.TestDataDirectory` more apparent. it's easy to confuse the two otherwise